docs/getting-started-guides/windows/index.md

```diff
@@ -1,139 +1,144 @@
 ---
 title: Windows Server Containers
 ---
+**Note:** These instructions were recently updated based on Windows Server platform enhancements
 
-Kubernetes version 1.5 introduces support for Windows Server Containers. In version 1.5, the Kubernetes control plane (API Server, Scheduler, Controller Manager, etc) continue to run on Linux, while the kubelet and kube-proxy can be run on Windows Server.
+Kubernetes version 1.5 introduced support for Windows Server Containers based on the Windows Server 2016 OS. With the release of Windows Server version 1709 and using Kubernetes v1.8 users are able to deploy a K8s cluster either on-premises or in a private/public cloud using a number of different network toplogies and CNI plugins. Platform improvements include:
+- Improved support for Pods! Shared network namespace (compartment) with multiple Windows Server containers (shared kernel)
+- Reduced network complexity by using a single network endpoint per Pod
+- Kernel-Based load-balancing using the Virtual Filtering Platform (VFP) Hyper-v Switch Extension (analogous to Linux iptables)
 
-**Note:** Windows Server Containers on Kubernetes is an Alpha feature in Kubernetes 1.5.
+ The Kubernetes control plane (API Server, Scheduler, Controller Manager, etc) continue to run on Linux, while the kubelet and kube-proxy can be run on Windows Server version 1709.
+
+**Note:** Windows Server Containers on Kubernetes is an Alpha feature in Kubernetes 1.8.
```

Comment:
It should be beta in Kubernetes 1.9 - please, update.

Reply:
@idvoretskyi  we have a pull on this ticket with the updated docs. please reference https://github.com/JMesser81/kubernetes.github.io/pull/1